digital_comms/ccam.py

```diff
@@ -223,9 +348,10 @@ def threshold_demand(self, SERVICE_OBLIGATION_CAPACITY):
 
     @property
     def demand(self):
-        """Estimate total demand based on population and penetration
+        """obj: The demand in capacity per km^2
 
-        E.g.
+        Notes
+        -----
             0.02 Mbps per user during busy hours
                 * 100 population
```

Comment:
I'm not sure I understand this example - double check.

Reply:
This example has been taken over from the original documentation. I will add a todo for the double check